food-truck-frontend/src/components/recommended-trucks/SearchRecommendedTrucksForm.tsx

```diff
@@ -64,6 +65,15 @@ class RecommendedTrucksForm extends Component<
     );
   }
 
+  getRouteLocations(trucks: RecommendedSimpleTruck[]) {
+      var routeLoc: any = [];
+      trucks.map((t, ndx) => {
+          let loc: any = backendToFrontend(t.loc,ndx);
+          routeLoc.push(loc);
+      })
+      return routeLoc
```

Comment:
For better type safety, you could use parse(RouteLocationMeta, backendToFrontend(t.loc, ndx))